vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -3232,19 +3226,15 @@ EXPORT(int, sceGxmProgramGetOutputRegisterFormat, const SceGxmProgram *program,
     return 0;
 }
 
-EXPORT(Ptr<SceGxmProgramParameter>, sceGxmProgramGetParameter, Ptr<const SceGxmProgram> program, uint32_t index) {
-    TRACY_FUNC(sceGxmProgramGetParameter, program, index);
+EXPORT(Ptr<SceGxmProgramParameter>, sceGxmProgramGetParameter, const Ptr<SceGxmProgram> program_ptr, uint32_t index) {
```

Comment:
```suggestion
EXPORT(Ptr<SceGxmProgramParameter>, sceGxmProgramGetParameter, Ptr<const SceGxmProgram> program_ptr, uint32_t index) {
```
It's the program which should not be modified, not the pointer to it